src/Groups/matrices/linear_isconjugate.jl

```diff
@@ -0,0 +1,155 @@
+# TODO: in this file are used many methods TEMPORARILY defined in files matrix_manipulation.jl and stuff_field_gen.jl
+# once methods in those files will be deleted / replaced / modified, this file need to be modified too
+
+
+
+export
+    generalized_jordan_block,
+    generalized_jordan_form,
+    isconjugate,
+    issemisimple,
+    isunipotent,
+    pol_elementary_divisors,
+    multiplicative_jordan_decomposition
+
+
+
+
+########################################################################
+#
+# Semisimple / Unipotent
+#
+########################################################################
+
+"""
+    multiplicative_jordan_decomposition(M::MatrixGroupElem)
+Return `S` and `U` in the group `G = parent(M)` such that `S` is semisimple, `U` is unipotent and  `M = SU = US`.
```

Comment:
Please add these empty lines everywhere:
```suggestion

Return `S` and `U` in the group `G = parent(M)` such that `S` is semisimple, `U` is unipotent and  `M = SU = US`.
```